### PR TITLE
feat(query): Added SQL Server Database With Unrecommended Retention Days for ARM

### PR DESCRIPTION
**Pull request by joaoReigota1**
Signed-off-by: João Reigota <joao.reigota@checkmarx.com>

**Proposed Changes**
- Added SQL Server Database With Low Retention Days for ARM

SQL Server Database Auditing Settings should keep the audit logs in the storage account for at least 90 days

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

File: assets/queries/azureResourceManager/sql_server_database_with_low_retention_days/query.rego [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+	value.type == types[_]
+	properties := value.properties
+	lower(properties.state) == "enabled"
+	not common_lib.valid_key(properties, "retentionDays")
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("%s.name=%s.properties", [common_lib.concat_path(path), value.name]),
+		"issueType": "MissingAttribute",
+		"keyExpectedValue": "'auditingSettings.properties.retentionDays' is definded and above 90 days",
```
Comment (rafaela-soares) [inline assets/queries/azureResourceManager/sql_server_database_with_low_retention_days/query.rego:21]: typo in `is definded` (same for line 42)

**Review by rafaela-soares — CHANGES_REQUESTED** [review]

File: assets/queries/azureResourceManager/sql_server_database_with_low_retention_days/metadata.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "id": "c09cdac2-7670-458a-bf6c-efad6880973a",
+  "queryName": "SQL Server Database With Low Retention Days",
```
Comment (rogeriopeixotocx) [inline assets/queries/azureResourceManager/sql_server_database_with_low_retention_days/metadata.json:3]:
```suggestion
  "queryName": "SQL Server Database With Inadequate Retention Days",
```

```suggestion
  "queryName": "SQL Server Database With Unrecommended Retention Days",
```

which one is better?

**Review by rogeriopeixotocx — CHANGES_REQUESTED** [review]
LGTM

**Review by rogeriopeixotocx — APPROVED** [review]
LGTM

**Review by rafaela-soares — APPROVED** [review]
LGTM

**Pull request merged**